content/ecosystem/bridges/xmpp/bridges.toml

```diff
@@ -25,7 +25,7 @@ maintainer = "LDA"
 summary = """
 The jealous XMPP-Matrix bridge
 """
-maturity = "Alpha"
+maturity = "Obsolete"
```

Comment:
parsee is not obsolete the main and only maintainer LDA has been really busy recently